client/src/components/App.jsx

```diff
@@ -41,13 +41,23 @@ class App extends React.Component {
         this.setState({reviews: response.data})
         // console.log('the state is now', this.state)
       })
+      .catch(function(error) {
+        if (!error.status) {
+          // network error
+        }
+      });
 
     axios.get('/api/users')
       .then((users) => {
         // console.log(users)
         this.setState({users: users.data})
         // console.log('the state is now', this.state)
       })
+      .catch(function(error) {
+        if (!error.status) {
+          // network error
+        }
+      });
 
   }
 
```

Comment:
Double Checking with you. Thought we were going to do functional react components with hooks. This is a class component.

Reply:
Hmmm you are correct. I think I need to spend this afternoon understanding hooks and refactor my existing work before moving on.